python/jsbeautifier/tests/core/test_options.py

```diff
@@ -102,15 +102,13 @@ def test__is_valid_selection(self):
 
     def test__get_selection_list(self):
         # should raise error with empty selection
-        with self.assertRaisesRegexp(
-            ValueError, "Selection list cannot" + " be empty."
-        ):
+        with self.assertRaisesRegex(ValueError, "Selection list cannot" + " be empty."):
```

Comment:
These are aliases that were retained for backward compat to python 2.x apparently.  In typical python style, they just suddenly stopped working on my system.  All versions of 3 support the modern names.